spot_rl_experiments/spot_rl/utils/a_star.py

```diff
@@ -71,10 +71,10 @@ def astar(grid, start, goal):
             )
 
             if is_area_free and is_area_within_bounds:
-                local_top = neighbor_center[0] - expand_range
-                local_bottom = neighbor_center[0] + expand_range
-                local_left = neighbor_center[1] - expand_range
-                local_right = neighbor_center[1] + expand_range
+                local_top = neighbor_center[0] - expand_range//2
```

Comment:
so the extent from the concept graph yaml is the full width and length of the bounding box?